addons/survey/tests/test_survey_invite.py

```diff
@@ -37,22 +37,27 @@ def test_survey_invite_action(self):
         action = self.survey.action_send_survey()
         self.assertEqual(action['res_model'], 'survey.invite')
 
-        # Bad cases
-        surveys = [
-            # no page
-            self.env['survey.survey'].create({'title': 'Test survey'}),
-            # no questions
-            self.env['survey.survey'].create({'title': 'Test survey', 'question_and_page_ids': [(0, 0, {'is_page': True, 'question_type': False, 'title': 'P0', 'sequence': 1})]}),
-            # closed
-            self.env['survey.survey'].with_user(self.survey_manager).create({
-                'title': 'S0',
+        bad_cases = [
```

Comment:
can you add "scoring_type != no_scoring" and "scoring_max_obtainable > 0" ?
(currently everything it raise the UserError, `no_scoring  !=  'no_scoring'` so the condition is not fully tested)